iModelCore/ECDb/ECDb/ECDbSqlFunctions.cpp

```diff
@@ -22,6 +22,7 @@ void PropExistsFunc::_ComputeScalar(Context& ctx, int nArgs, DbValue* args) {
         return;
     }
 
+    if (m_propMap.Empty()) m_propMap.Build(m_ecdb, true);

```

Comment:
put this on 2 lines and explain why it is lazy created rather than where it was before